### DISCUSSION TITLE
Comments disappearing, sometimes reappearing, showing for different users etc.

### DISCUSSION THREAD

**Discussion by cmd-s13** [outline/outline#7520] [Self Hosting] [answered]
Hi, we have been using outline for a few versions now. This has been happening since before 0.78.0. 

Every page functions differently with comments, some stay around forever with 100+ comments on the page whereas some completely disappear. Some users have reported them reappearing but not consistent. Resolved comments behave the same. Happens to new and old comments.

No error logs showing with debug on. No errors in console. Unsure of what could be wrong

**Comment by tommoor** [comment, marked as answer]
Most likely this was solved here, https://github.com/outline/outline/pull/7397 – fix in `0.79.0` soon
